src/transformers/models/beit/configuration_beit.py

```diff
@@ -139,14 +150,16 @@ def __init__(
         layer_scale_init_value=0.1,
         drop_path_rate=0.1,
         use_mean_pooling=True,
-        out_indices=[3, 5, 7, 11],
+        semantic_out_indices=[3, 5, 7, 11],
         pool_scales=[1, 2, 3, 6],
         use_auxiliary_head=True,
         auxiliary_loss_weight=0.4,
         auxiliary_channels=256,
         auxiliary_num_convs=1,
         auxiliary_concat_input=False,
         semantic_loss_ignore_index=255,
+        out_features=None,
+        out_indices=None,
```

Comment:
We can't make this breaking change - there's no warning for the users the `out_indices` isn't being used for the same thing anymore and old configs will load setting the backbone `out_indices` to whatever it was in the saved config